docs/sources/visualizations/panels-visualizations/visualizations/logs/index.md

```diff
@@ -85,14 +85,20 @@ Use these settings to refine your visualization:
 
 <!-- prettier-ignore-start -->
 
-| Option | Description |
-| ------ | ----------- |
+In the Panel options section of the panel editor pane, set basic options like panel title and description, as well as panel links.
```

Comment:
This is covered in the section above under the **Panel options** heading (line 78). It uses a shared content file that renders when you build the docs. 
```suggestion
```